### PR TITLE
convert tests to pytest

### PR DESCRIPTION
**Pull request by jimasuen**
Converted tests in test folder to pytest

### CODE REVIEW

File: tests/test_userwallet.py [hunk 1]
```diff
@@ -1,62 +1,105 @@
...
+        print(f"\nInvoice status: {res['paid']}. \n\nFull check invoice response: \n{res}")
 
-    async with ClientSession() as session:
-        uw = UserWallet(c, session)
-        # works
-        userwallet = await uw.get_wallet_details()
-        print(f"user wallet info : {userwallet}")
+    # pay invoice
+    async def test_pay_invoice(self):
+        bolt = "lnbc800n1ps23r2dpp5ahh77dmx2d6yd3cw2xheks20kwe3nwh4zhcllxzjcq5fat3kvkssdqsd9h8vmmfvdjk7mn9cqzpgrzjq02snzwz4petaly54yzjkm358rqa5as9hkgydjvxxmvlpuk6dfd9cz0y2cqq0qsqqyqqqqlgqqqqqqgq9qsp5cut63ftfcffwkrr2w9r50w5e40m93k3er75mc70ysxps7yercs9s9qyyssqs7qk3cz97nm5m6ehzedcxhttx87l7x5kk38gvwkzzv4lhrhddtqq3sk43nnvsddagf36ledw9vhlpqxuu5s53pj6sz926mwqxf8chsgp2m9j8w"
```
Comment (bitkarrot) [inline tests/test_userwallet.py:48]: not mandatory but might be good to an option to test these by asking for the bolt at runtime on cli when running the pytest

**Review by bitkarrot — APPROVED** [review]

**Comment by bitkarrot** [comment]
merged, thank you !

File: tests/test_userwallet.py (hunk 1 above)
Comment (jimasuen) [reply tests/test_userwallet.py:48]: This is a really good idea

Comment (jimasuen) [reply tests/test_userwallet.py:48]: Implemented this in: https://github.com/lightningames/pylnbits/pull/178

**Pull request merged**